src/r2mm/manager/linux/GameDirectoryResolver.ts

```diff
@@ -0,0 +1,198 @@
+import R2Error from '../../../model/errors/R2Error';
+import FileNotFoundError from '../../../model/errors/FileNotFoundError';
+import VdfParseError from '../../../model/errors/Vdf/VdfParseError';
+
+import * as vdf from '@node-steam/vdf';
+import * as path from 'path';
+import { homedir } from 'os';
+import ManagerSettings from '../ManagerSettings';
+import FsProvider from "../../../providers/generic/file/FsProvider";
+import GameDirectoryResolverProvider from '../../../providers/ror2/game/GameDirectoryResolverProvider';
+
+const appManifest = 'appmanifest_632360.acf';
+
+export default class GameDirectoryResolverImpl extends GameDirectoryResolverProvider {
```

Comment:
Would be good to rename the class to something like `LinuxGameDirectoryResolver` with the file name to match

Reply:
Same in the Windows variant too

Reply:
it's namespaced, so, theoretically, it's not needed

Reply:
It's more for IDEs referencing imports